File: SCALA/Cap07/c07ex03.scala
// c07ex03.scala

import scala.io.StdIn._

object Main {
  var A, B: Int = 0

  def troca(): Unit = {
    var X = A
    A = B
    B = X
  }

  def main(args: Array[String]): Unit = {
    print("Entre um valor para a variavel <A>: ")
    A = readInt()

    print("Entre um valor para a variavel <B>: ")
    B = readInt()

    troca()

    println()
    println(s"Variavel <A> com valor $A")
    println(s"Variavel <B> com valor $B")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: KOTLIN/Cap07/c07ex03.kt
// c07ex03.kt

var A: Int = 0
var B: Int = 0

fun troca() {
    var X = A
    A = B
    B = X
}

fun main() {
    print("Entre um valor para a variavel <A>: ")
    A = readLine()?.toIntOrNull() ?: 0

    print("Entre um valor para a variavel <B>: ")
    B = readLine()?.toIntOrNull() ?: 0

    troca()

    println()
    println("Variavel <A> com valor $A")
    println("Variavel <B> com valor $B")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
